packages/block-library/src/media-text/media-container.js

```diff
@@ -57,6 +66,19 @@ class MediaContainer extends Component {
 		);
 	}
 
+	renderImageFill() {
+		const { mediaUrl, className, focalPoint } = this.props;
+		const style = imageFillStyles( mediaUrl, focalPoint );
+		return (
+			<Fragment>
+				{ this.renderToolbarEditButton() }
+				<figure className={ className }>
+					<div style={ style } ></div>
```

Comment:
This is my only real concern. We now have a fundemental difference in the HTML we're outputting due to the need to "fill" the space (ie: it's not simple to have a HTML `img` tag fill a space as is required by this feature):

#### Markup _without_ Fill
```jsx
<figure class="wp-block-media-text__media">
	<img src="some-image.jpg">
</figure>
```

#### Markup _with_ Fill
```jsx
<figure class="wp-block-media-text__media">
	<div style="background-image:url(some-image.jpg);"></div>
</figure>
```

The use of an empty `<div>` provides no semantic meaning for users of assistive technologies. As far as they are concerned we've now got an empty `<figure>` which is potentially confusing. 

Under WCAG this is ok if the image does not form part of the "content" and is used for visual/design purposes only. However, as we have _no_ control of the context in which this feature will be used we cannot guarantee that the user will not utilise an image that is intended as part of the content (ie: without the image you lose some of the ability to comprehend the content itself).

Without a true `<img />` tag we lose all the a11y benefits it provides as a long standing citizen of the HTML spec.

To resolve this I'm going to suggest we rope in someone for a a11y review. My thoughts for what they are worth:

* Should we add a visually hidden `<img />` tag here which will be comprehensible to assistive technology?
* Should we add a `aria-hidden` role to the `<div>` in order to have assistive tech ignore it but allow it to remain visible for sighted users?
* Are there any established patterns/best practices around this?

Reply:
Would it work if we keep the same markup without fill and add the background style to the figure directly?
```
<figure class="wp-block-media-text__media is-fill" style="">
	<img src="some-image.jpg">
</figure>
.wp-block-media-text__media.is-fill {
display: none;
};
```

That seems the most semantic option.

Reply:
> it's not simple to have a HTML img tag fill a space as is required by this feature

This is actually just as simple as `background-img` https://github.com/WordPress/gutenberg/pull/14445#discussion_r267781903

https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit

Reply:
> Would it work if we keep the same markup without fill and add the background style to the figure directly?

@frontdevde Would that work? If not you could even used a pseudo element instead of the empty `<div>` and leave the `<img />` in place as suggested.

I don't think we can use `object-fit` as it's not supported in IE11 and the Focal Point picker isn't designed for this use case.